src/serialport.cpp

```diff
@@ -87,75 +87,74 @@ static void deleteQForFD(const int fd) {
 
 
 NAN_METHOD(Open) {
-  NanScope();
 
   // path
-  if(!args[0]->IsString()) {
-    NanThrowTypeError("First argument must be a string");
-    NanReturnUndefined();
+  if(!info[0]->IsString()) {
+    Nan::ThrowTypeError("First argument must be a string");
+    return;
   }
-  v8::String::Utf8Value path(args[0]->ToString());
+  v8::String::Utf8Value path(info[0]->ToString());
 
   // options
-  if(!args[1]->IsObject()) {
-    NanThrowTypeError("Second argument must be an object");
-    NanReturnUndefined();
+  if(!info[1]->IsObject()) {
+    Nan::ThrowTypeError("Second argument must be an object");
+    return;
   }
-  v8::Local<v8::Object> options = args[1]->ToObject();
+  v8::Local<v8::Object> options = info[1]->ToObject();
 
   // callback
-  if(!args[2]->IsFunction()) {
-    NanThrowTypeError("Third argument must be a function");
-    NanReturnUndefined();
+  if(!info[2]->IsFunction()) {
+    Nan::ThrowTypeError("Third argument must be a function");
+    return;
   }
-  v8::Local<v8::Function> callback = args[2].As<v8::Function>();
+  v8::Local<v8::Function> callback = info[2].As<v8::Function>();
 
   OpenBaton* baton = new OpenBaton();
   memset(baton, 0, sizeof(OpenBaton));
   strcpy(baton->path, *path);
-  baton->baudRate = options->Get(NanNew<v8::String>("baudRate"))->ToInt32()->Int32Value();
```

Comment:
It's not removed. It's replace with `Nan::Get`, which is said to be [compatible with all V8 versions](https://github.com/nodejs/nan/blob/master/doc/maybe_types.md#nanget).